src/Connector.SqlServer/Features/IBuildStoreDataForMode.cs

```diff
@@ -1,18 +1,20 @@
-﻿using System;
-using System.Collections.Generic;
-using CluedIn.Connector.SqlServer.Connector;
+﻿using CluedIn.Connector.SqlServer.Connector;
+using CluedIn.Connector.SqlServer.Utility;
 using CluedIn.Core;
 using CluedIn.Core.Data.Parts;
 using CluedIn.Core.Streams.Models;
 using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
 
 namespace CluedIn.Connector.SqlServer.Features
 {
     public interface IBuildStoreDataForMode
```

Comment:
Do not change a public interface.

There are three options here:

Go with a breaking change (Not allowed).
Default implementation (Not popular as it hides in the IDE's).
Extend the interface (I would do this, however any type checks would need to be updated).

Also SanitizedSqlName should be a string with the sanitization performed elsewhere.